src/catalog.h

```diff
@@ -353,10 +353,11 @@ enum Anum_chunk
 {
 	Anum_chunk_id = 1,
 	Anum_chunk_hypertable_id,
-	Anum_chunk_schema_name,
-	Anum_chunk_table_name,
 	Anum_chunk_compressed_chunk_id,
+	Anum_chunk_chunk_status,
 	Anum_chunk_dropped,
+	Anum_chunk_schema_name,
+	Anum_chunk_table_name,
 	_Anum_chunk_max,
 };
```

Comment:
I do not think it is a good idea to reorder the field names or add a new field in the middle, especially not in a minor release. There are too many thing that can go wrong.